### DISCUSSION TITLE
How to create a snapshot for Aurora Serverless V1

### DISCUSSION THREAD

**Discussion by ClaudenirFreitas** [aws/aws-cli#7575] [Q&A] [answered]
Hi all,
I have created a database using Aurora Serverless V1 but cannot create a snapshot via AWS CLI.

```bash
# command
      aws rds create-db-snapshot \
        --db-instance-identifier dummy-database \
        --db-snapshot-identifier snaphot \
        --region eu-west-1
        
# output 
An error occurred (DBInstanceNotFound) when calling the CreateDBSnapshot operation: DBInstance not found: dummy-database
```

When I got this error, I ran the command below:
```bash
# command
      aws rds describe-db-instances \
        --region eu-west-1
        
# output 
{
    "DBInstances": [
    ]
}
```

I did not find a solution for that... can you help me?

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
Hi @ClaudenirFreitas, thanks for reaching out.  Aurora Serverless V1 databases are created as DB clusters, instead of regular DB instances.  To create a snapshot or describe databases, you'd need to use the `create-db-cluster-snapshot` and `describe-db-clusters` methods.  Let me know if that doesn't work, or if you have any other questions.

**Reply by ClaudenirFreitas** [reply]
Cool, thanks a lot @RyanFitzSimmonsAK!  
It is working as expected.
